v2/kafka-to-kafka/src/main/java/com/google/cloud/teleport/v2/templates/KafkaToKafka.java

```diff
@@ -82,7 +82,10 @@ public static PipelineResult run(KafkaToKafkaOptions options) {
       checkArgument(
           options.getKafkaReadKeyPasswordSecretId().trim().length() > 0,
           "KafkaReadKeyPasswordSecretId version for key password required for SSL authentication");
-    } else if (options.getKafkaReadAuthenticationMode().equals(KafkaAuthenticationMethod.NONE)) {
+    } else if (options.getKafkaReadAuthenticationMode().equals(KafkaAuthenticationMethod.NONE)
```

Comment:
nit: I'd add a comment explaining why the if block is empty (e.g. "no extra validation needed for these auth mechanisms as they don't require any extra parameters").

Reply:
Done